src/lib/y2journal/query_presenter.rb

```diff
@@ -96,6 +96,11 @@ def interval_description
       end
     end
 
+    # Split journalctl timestamp at "—"; localize and return the joined string back
```

Comment:
This description is not accurate. It looks like `ts` is a string with the dates separated by "-", which is not the case. Even the real behavior (a class method receiving an array of timestamps) looks weird to me. It should likely just receive a `Boot` object.

I would prefer something like:

```ruby 
def boot_timestamps(boot)
  first = boot[:timestamps][:first]
  last = boot[:timestamps][:last]

  #TRANSLATORS: %{start_time} and %{end_time} will be replaced by already localized timestamps indicating
  # when the system booted and when it was shut down
  _("%{start_time} to %{end_time}") % {start_time: first, end_time: last}
end
```

Then the call would be more straightforward and meaningful:

```ruby
boot_timestamps(boots.last)
```

Reply:
I've made the relevant changes and pushed a new commit.